### PR TITLE
Remove '/posts' route

### PR DESCRIPTION
**Pull request by Hives**

### CODE REVIEW

**Comment by Hives** [comment]
[Remove '/posts' route](https://trello.com/c/UQ5gGLCb/77-remove-posts-route)

**Comment by j-dav** [comment]
Looks fine, waiting for 2nd person to review

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -35,24 +32,22 @@ def destroy
   end
 
   def create
-    logged_in_user = current_user
-    params = post_params
-    params[:author_id] = logged_in_user.id
-    @post = Post.create(params)
-    recipient = User.find(params[:recipient_id])
+    @post = Post.create(post_params.merge(author_id: current_user.id))
+    recipient = User.find(post_params[:recipient_id])
```
Comment (j-dav) [inline app/controllers/posts_controller.rb:36]: merge?

**Comment by arthurashman** [comment]
Looks good. 

Need to find out where recipient id gets lost on updating a post and why.

**Pull request merged**